setup-environment-internal

```diff
@@ -262,22 +262,39 @@ ln -sf "${MANIFESTS}" "${OEROOT}"/layers/
 
 DISTRO_DIRNAME=$(echo "${DISTRO}" | sed 's#[.-]#_#g')
 
+LMP_TAG="$(git --git-dir ${MANIFESTS}/.git describe HEAD --tags --abbrev=0)"
+LMP_TAG="$(echo $LMP_TAG | sed 's/\.[0-9]*$//')"
```

Comment:
We should add a comment above here explaining that this is stripping the decimal from a potential point release.

Reply:
added the comment, copy-pasted description from the commit message that introduced this line